### ISSUE TITLE
Suggestion: Toggle flag explosions based on online Town members.

### ISSUE THREAD

**Issue by Darutan92** [TownyAdvanced/Towny#3711]
Players would like to use TNT guns and plugin Cannons (https://www.spigotmc.org/resources/cannons.56764/) in a siege of cities, wars and raids, but the flag can be turned off if desired. Forced inclusion of the flag leads to abuse of explosions by dishonorable players.

You can do auto-switching based on online (if 1 member is online (or how much you set up) - explosions in the city work).

**Comment by Goosius1** [comment]
Speaking to the siegewar system, thats certainly a good idea, one trial server has already modded it in.

However, I assume you are using one of the two released systems.

Just to help you get your ticket to the right inbox, which system are you looking for the update in? - event war or flag war?

**Comment by Darutan92** [comment]
Hey. We use "flag war" or plugin "WFT".

**Comment by LlmDl** [comment]
Towny's in-built FlagWar or WarsForTowny? They are not the same.

**Comment by Darutan92** [comment]
> Towny's in-built FlagWar or WarsForTowny? They are not the same.

Yes, I understand, but their work is similar. We use in-built FlagWar.

**Comment by LlmDl** [comment]
This might end up being one of those settings that works into the amalgamation of war settings that's being worked on, that way numerous war types could take advantage of it.
